easybuild/easyconfigs/v/vitessceR/vitessceR-20230110-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,67 @@
+easyblock = 'Bundle'
+
+name = 'vitessceR'
+# the github repo has no releases / tags, that's why we have to use commit...
+version = '20230110'
+versionsuffix = '-R-%(rver)s'
+local_commit = '8c67168'
+
+homepage = 'https://github.com/vitessce/vitessceR'
+description = "Vitessce is a visual integration tool for exploration of spatial single-cell experiments."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('R', '4.2.1'),
+    ('R-bundle-Bioconductor', '3.15', versionsuffix),
+    ('libsodium', '1.0.18'),
+]
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/',
+        'https://cran.rstudio.com/src/contrib/',
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s/',
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+exts_list = [
+    ('webutils', '1.1', {
+        'checksums': ['33e2b408d2b9ca12e158e0007d4a55a3ace6838341ad944344de2ceb806e4699'],
+    }),
+    ('sodium', '1.2.1', {
+        'checksums': ['f76e98969710af8b6c7c7decf63706d91185eebf8a266475b554d5ad95f803a8'],
+    }),
+    ('swagger', '3.33.1', {
+        'checksums': ['528369b04c6142112e05ee8b950ca1c01b3e390a4515eacb0fa17db98bafafd1'],
+    }),
+    ('plumber', '1.2.1', {
+        'checksums': ['6ffc13e5ce1ff7ec81f6a4ab04eb1cb9f6da4004cf205ba3098d2ec4a83f1ecc'],
+    }),
+    ('varhandle', '2.0.5', {
```

Comment:
@deniskristak Is there any reason we shouldn't add these to `R-4.2.1-foss-2022a.eb` instead?

We shouldn't do that for `vitessceR`, since that's not provided via PyPI

Reply:
probably not, adding